go/pkg/cli/checkout_test.go

```diff
@@ -118,4 +121,25 @@ func TestCheckout(t *testing.T) {
 	contents, err = ioutil.ReadFile(path.Join(outputDir2, rand2))
 	require.NoError(t, err)
 	require.Equal(t, rand2, string(contents))
+
+	outputDir3, err := files.TempDir("test-checkout-output-3")
```

Comment:
Could we have two more test cases:
* Checking out a directory
* Checking out a single file inside a subdirectory

Reply:
Added both cases.